### PR TITLE
MDCT-1979: Delete Measure Details

### PR DESCRIPTION
**Pull request by ntsummers1**
## Description
A measured response for a measured entity
Fixes [MDCT-1979](https://qmacbis.atlassian.net/browse/MDCT-1979)

https://user-images.githubusercontent.com/19439679/193901451-1a2103b3-d8e9-4ff6-9039-43cd06e8aec1.mov

### How to test
./dev local
Create a program
Enter Program
Go to mcpar/program-level-indicators/availability-and-accessibility/access-measures
Add a few access measures
Delete however many you want!

## Code author checklist
- [x] I have performed a self-review of my code
- [x] I have added [thorough](https://bit.ly/3zPrxuZ) tests
- [x] I have added analytics, if necessary
- [x] I have updated the documentation, if necessary

## Reviewer checklist (two different people)
- [ ] I have done the deep review and verified the items in the above checklist are g2g
- [ ] I have done the lgtm review

### CODE REVIEW

File: services/ui-src/src/components/modals/DeleteEntityModal.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      submitting={deleting}
+      content={{
+        heading: `Delete access measure?`,
+        actionButtonText: `Yes, Delete Measure`,
+        closeButtonText: "Cancel",
+      }}
+    >
+      <Text>
+        You will lose all information entered for this measure. Are you sure you
+        want to proceed
```
Comment (braxex) [inline services/ui-src/src/components/modals/DeleteEntityModal.tsx:52]: Typo: Guessing there's supposed to be a `?` at the end.

**Review by braxex — APPROVED** [review]
Looks good. Left a few comments/suggestions but nothing blocking. 👍

File: services/ui-src/src/components/modals/DeleteEntityModal.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
+// components
+import { Text } from "@chakra-ui/react";
+import { Modal, ReportContext } from "components";
+import { useContext, useState } from "react";
+// types
+import { AnyObject, EntityShape, ReportStatus } from "types";
+import { useUser } from "utils";
+
+export const DeleteEntityModal = ({
```
Comment (gmrabian) [inline services/ui-src/src/components/modals/DeleteEntityModal.tsx:9]: This feels so similar to DeleteDynamicFieldRecordModal that I wonder if they could be combined. Like pass in the desired function ala`removeRecord`, pass in the desired text

Comment (braxex) [reply services/ui-src/src/components/modals/DeleteEntityModal.tsx:9]: Hmm... they are very similar. I'm thinking for now let's keep them separate and review after we finalize the ModalDrawerReportPage changes through access measures, quality measures, and sanctions. There may yet be changes we're not anticipating.

**Pull request merged**